modules/social_features/social_group/social_group.install

```diff
@@ -640,3 +642,74 @@ function social_group_update_8906() {
     $config->save(TRUE);
   }
 }
+
+/**
+ * Helper function to import configurations.
+ *
+ * @param array $configs
+ *   Array of configuration file names and their folders.
+ */
+function _social_group_features_import_config_helper(array $configs) {
```

Comment:
Move this `social_core` module and make it a service class. Also, make this function a static function.

Reply:
Moved.